File: Civitas-Java/Civitas/src/main/java/civitas/OperacionInmobiliaria.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package civitas;

/**
 * @author elena
 * @date 25/10/2019
 */
public class OperacionInmobiliaria {
    private int numPropiedad;
    private GestionesInmobiliarias gestion;
    
    public OperacionInmobiliaria(GestionesInmobiliarias gest, int ip){
        numPropiedad = ip;
        gestion = gest;
    }
    
    public GestionesInmobiliarias getGestion(){ return gestion; }    
    public int getNumPropiedad(){ return numPropiedad; }
}

File: Civitas-Ruby/lib/operacion_inmobiliaria.rb
# To change this license header, choose License Headers in Project Properties.
# To change this template file, choose Tools | Templates
# and open the template in the editor.

module Civitas
  class OperacionInmobiliaria
    def initialize(gest, ip)
      @num_propiedad = ip
      @gestion = gest
    end
    
    attr_reader :num_propiedad, :gestion
    
  end
end
